out/.env

```diff
@@ -0,0 +1,8 @@
+TARGET_OUT_LINUX=/usr/local/google/home/gargnupur/go/src/istio.io/proxy/out/linux_amd64
```

Comment:
accident commit?

Reply:
hehe yes.. working on too many pr's at the same time :(